.travis.yml

```diff
@@ -0,0 +1,15 @@
+---
+language: python
+python:
+  - "2.7"
+  - "3.6"
+install:
+  - gem install mdl
+  - pip install rstcheck
+  - pip install sphinx
+script:
+  # Linting
+  # https://github.com/markdownlint/markdownlint
+  - mdl -s relaxed README.md
```

Comment:
Use something like `markdownlint --ignore <the things> README.md` if we're sticking to python.